.github/workflows/ci.yml

```diff
@@ -793,41 +720,14 @@ jobs:
           asset_path: ./rizin-src/rizin-src.tar.xz
           asset_name: rizin-src-${{ steps.extract_version.outputs.branch }}.tar.gz
           asset_content_type: application/zip
-    - name: Upload rizin debian:buster package
-      uses: actions/upload-release-asset@v1
-      env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: ./rizin-debian-buster-deb/rizin_amd64.deb
-          asset_name: rizin-debian-buster_${{ steps.extract_version.outputs.branch }}_amd64.deb
-          asset_content_type: application/zip
-    - name: Upload rizin-dev debian:buster package
-      uses: actions/upload-release-asset@v1
-      env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: ./rizin-dev-debian-buster-deb/rizin-dev_amd64.deb
-          asset_name: rizin-dev-debian-buster_${{ steps.extract_version.outputs.branch }}_amd64.deb
-          asset_content_type: application/zip
-    - name: Upload rizin ubuntu:18.04 package
-      uses: actions/upload-release-asset@v1
-      env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: ./rizin-ubuntu-1804-deb/rizin_amd64.deb
-          asset_name: rizin-ubuntu-1804_${{ steps.extract_version.outputs.branch }}_amd64.deb
-          asset_content_type: application/zip
-    - name: Upload rizin-dev ubuntu:18.04 package
+    - name: Upload rizin static x86_64 tar.gz
       uses: actions/upload-release-asset@v1
       env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
           upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: ./rizin-dev-ubuntu-1804-deb/rizin-dev_amd64.deb
-          asset_name: rizin-dev-ubuntu-1804_${{ steps.extract_version.outputs.branch }}_amd64.deb
+          asset_path: ./rizin-static/rizin-static.tar.xz
+          asset_name: rizin-${{ steps.extract_version.outputs.branch }}-static-x86_64.tar.gz
```

Comment:
why the conversion from tar.xz to tar.gz?

Reply:
No conversion, forgot to change.